featuretools/primitives/standard/cum_transform_feature.py

```diff
@@ -5,8 +5,20 @@
 
 
 class CumSum(TransformPrimitive):
-    """Returns the cumulative sum after grouping"""
-
+    """Calculates the cumulative sum over a grouping.
```

Comment:
no need to say "over a grouping" for all these methods. the "over a grouping" is from an older version of featuretools